lib/src/instance.dart

```diff
@@ -793,8 +793,8 @@ class VMTypedDataInstance<T extends TypedData> extends VMTypedDataInstanceRef<T>
 
   static TypedData _value(Map json) {
     var byteList = BASE64.decode(json["bytes"]);
-    var bytes = byteList is TypedData
-        ? byteList
+    TypedData bytes = byteList is TypedData
+        ? byteList as TypedData
```

Comment:
Why is this cast necessary? The `is` check should propagate its type information here.